### ISSUE TITLE
Accessibility: ListViewGroup Collapsible/Collapsed

### ISSUE THREAD

**Issue by lonitra** [dotnet/winforms#3269]
**Problem description:**
A Collapsible ListViewGroup cannot be collapsed/expanded via keyboard only. Additionally, there is no Narrator announcement for the collapse/expand action.

**Expected behavior:**
User should be able to collapse/expand a collapsible ListViewGroup via keyboard only, perhaps by pressing the SPACE bar. 

Narrator should also announce the expand/collapse action. Something similar to what the PropertyGrid does when something is collapsed or expanded:

![image](https://user-images.githubusercontent.com/30007367/81727214-79775100-943d-11ea-9f34-4519c848a1a1.png)

![image](https://user-images.githubusercontent.com/30007367/81727101-49c84900-943d-11ea-8f85-d572f06dec44.png)

**Minimal repro:**

1. Create a winforms core app
2. Add a ListView and ListViewGroup 
3. Set Collapsible = true for ListViewGroup
4. Run app
5. Try to expand/collapse ListViewGroup via keyboard only
6. Start Narrator and expand/collapse through clicks

**Note:**
This stems from TODO in PR #3155

**Comment by hughbe** [comment]
`IExpandCollapseProvider` seems perfect for this

**Comment by RussKie** [comment]
> User should be able to collapse/expand a collapsible ListViewGroup via keyboard only, perhaps by pressing the SPACE bar.

I think <kbd>SPACE</kbd> is more associated with "enact" (e.g. similar to <kbd>ENTER</kbd>, _press button_), for expand/collapse - <kbd>←</kbd> <kbd>→</kbd> are typically used (e.g. in Windows Explorer or in a TreeView control), so I think we should use those.



> `IExpandCollapseProvider` seems perfect for this

Interesting idea 👍 I let @M-Lipin to comment on this.

**Comment by weltkante** [comment]
putting the suggestion in relation with other controls:
* checkboxes and toggle buttons are activated by space
* WPF has a standalone expand/collapse control ("expander") where the header can be focused as a whole and is activated (toggled) by space
* treeview expand/collapse is activated by arrow keys, however you focus the item and cannot focus the expand glyph
